symphony/lib/core/class.configuration.php

```diff
@@ -28,6 +28,12 @@ class Configuration
     private $_forceLowerCase = false;
 
     /**
+     * The string representing the tab characters used to serialize the configuration
+     * @var string
+     */
+    const TAB = '    ';
```

Comment:
`\t` has better cross platform support (although really we should probably move to space)